samples/WOCCatalog/WOCCatalog/PickersViewController.m

```diff
@@ -23,31 +23,22 @@ - (void)viewDidLoad {
     
     self.view.autoresizesSubviews = TRUE;
     self.view.backgroundColor = [UIColor whiteColor];
+    [self tableView].separatorColor = [UIColor clearColor];
     [self tableView].allowsSelection = NO;
+    [self tableView].scrollEnabled = NO;
 
     self.title = @"Pickers";
     
     // PICKER VIEW
-    
-    self.datePickerView = [[UIDatePicker alloc] initWithFrame:CGRectZero];
+    self.datePickerView = [[UIDatePicker alloc] initWithFrame:[self calcFrameForPicker]];
```

Comment:
If we're overriding the size later, there's no point in setting it here.
If there is, something's broken :)